ui-participant/src/hub/HubPage.tsx

```diff
@@ -179,9 +179,14 @@ function TaskGrouping({ title, tasks, enrollee, studyShortcode }: {
   title: string, tasks: ParticipantTask[],
   enrollee: Enrollee, studyShortcode: string
 }) {
+  const hasLockedTasks = tasks.some(task => !isTaskAccessible(task, enrollee))
+
   return (
     <>
       <h2 className="fs-6 text-uppercase mb-0">{title}</h2>
+      {hasLockedTasks && (
+        <p className="my-2">Some surveys may be locked until other required tasks are completed.</p>
```

Comment:
The design says "may be", but since this is only displayed when there are locked tasks, maybe change this to "are"?